src/views/leaderboard/Hero/index.tsx

```diff
@@ -89,7 +88,6 @@ const StyledCenterContainer = styled.div`
   flex-direction: column;
   align-items: center;
   right: 10%;
-  z-index: 1000;
```

Comment:
Because of this

![CleanShot 2022-07-20 at 09 16 28@2x](https://user-images.githubusercontent.com/1924968/179874765-48bc939c-73be-459e-add7-345619849d1c.jpg)

Reply:
got it, just fixed the issue